### DISCUSSION TITLE
Batch effect correction for differential expression between conditions

### DISCUSSION THREAD

**Discussion by plenehan** [satijalab/seurat#4564] [Q&A] [answered]
I see that in the Vignette for data integration (https://satijalab.org/seurat/articles/integration_introduction.html), it indicates that we should use the 'Integrated' assay for dimensionality reduction and clustering but then switch back to the 'RNA' assay before proceeding with differential expression analyses. Does this mean that the integration is only being used to help with clustering and dimensionality reduction visualization? Is there a reason that batch correction would not be important for the differential expression part of such a workflow? And is there a way to continue correcting for batch effects when performing differential expression analyses?

**Comment by timoast** [comment, marked as answer]
Please see https://github.com/satijalab/seurat/discussions/4000
